packages/editor/src/store/selectors.js

```diff
@@ -1723,7 +1725,10 @@ export function __experimentalGetTemplateInfo( state, template ) {
 
 	const templateTitle = isString( title ) ? title : title?.rendered;
 	const templateDescription = isString( excerpt ) ? excerpt : excerpt?.raw;
-	const templateIcon = getTemplatePartIconByArea( area );
+	const templateIcon =
+		__experimentalGetDefaultTemplatePartAreas( state ).find(
+			( item ) => area === item.area
+		)?.icon || layout;
```

Comment:
Shouldn't we use `getTemplatePartIcon` here? Something like this:
```suggestion
	const templateIcon =
		getTemplatePartIcon( __experimentalGetDefaultTemplatePartAreas( state ).find(
			( item ) => area === item.area
		)?.icon );
```

Reply:
`__experimentalGetDefaultTemplatePartAreas` already uses `getTemplatePartIcon`, so in that case we would be sending an already found icon back through the function which in turn wouldn't map correctly and return everything as the default icon 'layout'.

Reply:
Gotcha!